hello.py

```diff
@@ -40,10 +40,10 @@ def kill_server():
 
 @app.route('/autocomplete', methods=['GET'])
 def autocomplete():
-    search = request.args.get('q')
+    search = request.args.get('q').upper()
```

Comment:
Don't you need to update the template as well?

Reply:
Yessssss.... 😊